src/app/core/parsers/zod-rundown-event-parser.service.spec.ts

```diff
@@ -192,12 +202,12 @@ describe(ZodRundownEventParser.name, () => {
     it('does not parse a partial rundown infinite piece added event', () => {
       const mockedEntityParser = createMockOfEntityParser()
       const testee = new ZodRundownEventParser(instance(mockedEntityParser))
-      const event: Partial<RundownInfinitePieceAddedEvent> = {
-        type: RundownEventType.INFINITE_PIECE_ADDED,
+      const event: Partial<RundownInfinitePiecesUpdatedEvent> = {
+        type: RundownEventType.INFINITE_PIECES_UPDATED,
         rundownId: 'some-rundown-id',
       }
 
-      const result = (): RundownInfinitePieceAddedEvent => testee.parseInfinitePieceAddedEvent(event)
+      const result = (): RundownInfinitePiecesUpdatedEvent => testee.parseInfinitePieceAddedEvent(event)
```

Comment:
`.parseInfinitePieceAddedEvent` should also be updated.

Reply:
Updated!